src/panels/Scene/SceneTree/SceneTree.tsx

```diff
@@ -71,6 +77,7 @@ export function SceneTree() {
       <Group justify={'space-between'} gap={'xs'} mr={'xs'}>
         <FilterList.InputField placeHolderSearchText={'Search for a node...'} flex={1} />
         <SceneTreeFilters setFilter={setFilter} filter={filter} />
+        <FilterList.SearchSettingsMenu keys={allowedKeys} setKey={toggleKey} />
```

Comment:
I would probably put this before the filter, since this is more closely related to the search field that the filter settings are